source/val/validate_memory.cpp

```diff
@@ -1276,14 +1276,13 @@ spv_result_t ValidateCopyMemory(ValidationState_t& _, const Instruction* inst) {
 
 spv_result_t ValidateAccessChain(ValidationState_t& _,
                                  const Instruction* inst) {
-  std::string instr_name =
```

Comment:
moved string logic to error message time as assume most `OpAccessChain`  are valid and don't need to spend time building this string